cmd/provenanced/config/toml.go

```diff
@@ -0,0 +1,70 @@
+package config
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"text/template"
+
+	"github.com/spf13/viper"
+)
+
+const defaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
```

Comment:
Super nit: right alignment. :eye-twitch:

Reply:
most of the stuff in the config folder was directly from the SDK upstream... could use some polish